shakenfist/tasks.py

```diff
@@ -99,6 +99,25 @@ def error_msg(self):
         return self._error_msg
 
 
+class SnapshotInstanceTask(InstanceTask):
+    _name = 'instance_snapshot'
+
+    def __init__(self, instance_uuid, snap_uuid):
+        super(InstanceTask, self).__init__()
+        self._instance_uuid = instance_uuid
+        self.snap_uuid = snap_uuid
+
+        self._network = []
+        # General checks
+        if not instance_uuid:
+            raise NoInstanceTaskException(
+                'No instance specified for InstanceTask')
+        if not isinstance(instance_uuid, str):
+            raise NoInstanceTaskException('Instance UUID is not a string')
+        if not isinstance(self._network, list):
+            raise NetworkNotListTaskException()
```

Comment:
There is no network here?

Reply:
That comes from being copied and pasted. I feel like I've been a bit of a potato about the entire init thing, given that the requisite logic is in the `__init__` for `InstanceTask` which I should already be calling, but instead I'm calling `__init__` for `QueueTask`. Let me unbreak that now